src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -150,6 +181,32 @@ class TextSentimentSuiteV4 extends TestBase with DataFrameEquality with TextKey
       data(0).get(2).toString == "negative")
     assert(data(1).get(0).toString == "positive")
   }
+
+  lazy val df2: DataFrame = Seq(
+    ("en", "Hello world. This is some input text that I love."),
+    ("fr", "Bonjour tout le monde"),
+    ("es", "La carretera estaba atascada. Había mucho tráfico el día de ayer."),
+    ("es", "La carretera estaba atascada."),
+    ("es", "Había mucho tráfico el día de ayer."),
+    ("es", "La carretera estaba atascada. Había mucho tráfico el día de ayer."),
+    (null, "ich bin ein berliner")
+  ).toDF("lang", "text")
+
+  test("Sentiment - batch usage"){

```

Comment:
what is the purpose of this testing case here? if we would like to test the current default batch setup, then we might want to add some assertions here to show it actually batched according to the default value? Let's say we have 5 as default, then the input is 7, the batched result will be two rows of 5 first and 2 second? something like this?